src/SVGSkin.js

```diff
@@ -60,6 +70,34 @@ class SVGSkin extends Skin {
         super.setRotationCenter(x - viewOffset[0], y - viewOffset[1]);
     }
 
+    /**
+     * Create a MIP for a given scale and pass it a callback for updating
+     * state when switching between scales and MIPs.
+     * @param {number} scale - The relative size of the MIP
+     * @return {SVGMIP} An object that handles creating and updating SVG textures.
+     */
+    createMIP (scale) {
+        const callback = textureData => {
+            if (scale > this._largestTextureScale) {
+                this._largestTextureScale = scale;
+                // Currently silhouette only gets scaled up
+                this._silhouette.update(textureData);
+            }
+
+            if (scale === 1) {
```

Comment:
This seems a bit flaky-- it looks like it assumes `createMIP` will first be called with scale 1, which may not be the case. Could this be moved elsewhere, perhaps to `setSVG`?

Reply:
Hmm, it's not meant to assume `createMIP` will be called with a scale 1. The intention instead is _if_ it gets called with a scale of 1, use the `textureData` to set the `_maxTextureScale` and update the silhouette. We need it to be in the callback in `createMIP` instead of `setSVG` because we need the `textureData` that is passed to the callback in `SVGMIP.draw`.

Reply:
I did this sequence of steps and this one weird trick happened
![line cat](https://user-images.githubusercontent.com/2855464/65554851-c29f8280-def8-11e9-8740-9e919865348e.gif)

Reply:
This thing I saw may have been https://github.com/LLK/scratch-gui/issues/5215